### PR TITLE
Fix for issue #1882

### PR DESCRIPTION
**Pull request by chearius**
Fix for issue #1882 - No upstream DNS for dnsmasq for static IP configuration.

### CODE REVIEW

**Comment by detiber** [comment]
@sdodson thoughts compared to your approach here?

**Comment by sdodson** [comment]
I think we can use IP4_NAMESERVERS in all cases, can we not? see https://github.com/openshift/openshift-ansible/pull/1875

File: roles/openshift_node_dnsmasq/files/networkmanager/99-origin-dns.sh [hunk 1]
```diff
@@ -43,9 +43,19 @@ EOF
...
     echo '' > /etc/dnsmasq.d/origin-upstream-dns.conf
-    for ns in ${DHCP4_DOMAIN_NAME_SERVERS}; do
-       echo "server=${ns}" >> /etc/dnsmasq.d/origin-upstream-dns.conf
-    done
+
+    if [ -n "${DHCP4_DOMAIN_NAME_SERVERS}" ]; then
+        for ns in ${DHCP4_DOMAIN_NAME_SERVERS}; do
+            echo "server=${ns}" >> /etc/dnsmasq.d/origin-upstream-dns.conf
+        done
+    fi
```
Comment (sdodson) [inline roles/openshift_node_dnsmasq/files/networkmanager/99-origin-dns.sh]: Lets just get rid of this block and then I'm fine with taking this PR over mine.

**Comment by sdodson** [comment]
@chearius thanks for the PR, we've gone ahead and merged a fix for this already incorporating some improvements from your PR. If you're still having problems let us know.

**Comment by moander** [comment]
This PR solved my problem. Thank you very much!

**Comment by detiber** [comment]
Closing since #1875 provides a similar fix.

**Pull request closed without merging**